.env.example

```
@@ -0,0 +1,27 @@
+CAS_S3_REGION=                   # region substring of the endpoint, e.g. us-west-004
+CAS_S3_BUCKET=                   # private B2 bucket for chunks (B2 → Buckets → Create a Bucket, Private)
+CAS_S3_ACCESS_KEY_ID=            # B2 keyID (Application Keys → Add a New Application Key, bucket-scoped, Read+Write)
+CAS_S3_SECRET_ACCESS_KEY=        # B2 applicationKey (shown once on key creation)
+CAS_CHUNK_PREFIX=chunks/         # object-key prefix for chunk store (optional; default chunks/)
+CAS_INDEX_PREFIX=indexes/        # object-key prefix for artifact indexes (optional; default indexes/)
+
+# ── Ingest (tus resumable upload) ─────────────────────
+INGEST_UPLOAD_DIR=               # local disk dir for in-progress tus uploads
+INGEST_MAX_BYTES=5368709120      # max upload size in bytes (default ~5 GiB)
+INGEST_TUS_URL=                  # public ingest-tus origin, e.g. https://ingest.example.com
+INGEST_ALLOWED_ORIGIN=           # optional browser origin allowed to call ingest-tus, e.g. https://app.example.com
```

dotenv-linter (4.0.0)

[warning] [SpaceCharacter] The line has spaces around equal sign

(SpaceCharacter)